otter/check/notebook.py

```diff
@@ -401,7 +400,12 @@ def export(self, nb_path=None, export_path=None, pdf=True, filtering=True, pageb
             self._logger.debug(f"Added .otter file to zip file: {dot_otter}")
 
         for file in files:
-            zf.write(file)
+            if os.path.isdir(file):
+                sub_files = glob(f"./{file}/*")
+                for sub_file in sub_files:
+                    zf.write(sub_file)
```

Comment:
It doesn't look like this handles directories inside the `file` directory correctly -- I think it will cause an error if one is encountered.

Reply:
Updated in the new commits.